tests/ml/ml_test.py

```diff
@@ -536,7 +536,12 @@ def test_kbinsdiscretizer(tmpdir, strategy):
                                y=[1, 2, 5, 6, 10, 9])
 
     trans = vaex.ml.KBinsDiscretizer(features=['x', 'y'], n_bins=3, strategy=strategy)
-    df_train_trans = trans.fit_transform(df_train)
+
+    if strategy == 'quantile':
+        with pytest.warns(UserWarning, match='Bins whose width are too small'):
```

Comment:
For some reason, this doesn't trigger on Linux, and causes CI to fail.

Reply:
yeah i tested it also, and on a mac it also does not show a warning anymore.. maybe related to a numpy version.. Anyway now removed, but I found some other warnings to get rid off :)